lib/util.js

```diff
@@ -181,9 +181,10 @@ inspect.colors = {
   'red': [31, 39],
   'yellow': [33, 39]
 };
+inspect.colors = Object.assign(Object.create(null), colors);
```

Comment:
Can this be simplified to

```js
inspect.colors = Object.assign(Object.create(null), {
  'bold': //...
});
```

Doesn't seem like `colors` and `styles` are likely to be used elsewhere.